scripts/unhcr_data.py

```diff
@@ -144,6 +144,11 @@ def clean_hcr_data_download(df: pd.DataFrame) -> pd.DataFrame:
     return df.astype({"Individual refugees from Ukraine recorded across Europe": int})
 
 
+def save_hcr_data(df: pd.DataFrame) -> None:
+    """Save output as a CSV in output folder"""
+    df.to_csv(r"../output/hcr_data.csv")
```

Comment:
We never do relative paths.

Please change this to
`f"{config.PATHS.output}/hcr_data.csv"`